x/qgb/types/valset_confirm.go

```diff
@@ -9,21 +9,6 @@ import (
 
 var _ sdk.Msg = &MsgValsetConfirm{}
 
-// NewMsgValsetConfirm returns a new msgValSetConfirm.
-func NewMsgValsetConfirm(
-	nonce uint64,
-	ethAddress common.Address,
-	validator sdk.AccAddress,
-	signature string,
-) *MsgValsetConfirm {
-	return &MsgValsetConfirm{
-		Nonce:        nonce,
-		Orchestrator: validator.String(),
-		EthAddress:   ethAddress.Hex(),
-		Signature:    signature,
-	}
-}
```

Comment:
[blocking][question]
We'll still want to create these tx types, no?

Reply:
I  thought maybe we want to take this to the orchestrator/relayer side and remove  anything that is not explicitly used in the state machine. What do you think?

Reply:
Hmmm... perhaps!! if we do that, then we have to entirely remove methods to msgServer that handle those types.

Reply:
yeah, I agree after thinking about it for a few minutes lol